employees/tests/test_unit_report_model.py

```diff
@@ -84,6 +82,13 @@ def test_report_model_editable_field_should_have_default_value(self):
     def test_report_model_last_update_field_should_be_changed_on_update(self):
         self.field_auto_now_test("last_update", "description", "Updated")
 
+    @mock.patch("employees.models.Report")
+    def test_full_clean_method_should_be_called_on_save(self, mocked_model):
```

Comment:
You shouldn't mock whole Report and you don't really need to write test for every class which will use new model once it only inherent from only this class. :)

Reply:
I mean it's not bad, but it actually is unnecessary :P If you would test only once if it's working it would be fine. The only thing is that you're patch every time exact model instead of `full_clean` method. Please change it to mock `full_clean` ;)